### PR TITLE
Crux fixes

### PR DESCRIPTION
**Pull request by minhaz**
- Fixed url case sensitivity issue
- Added schema descriptions
- Added pptx file for diagram
- Added isadminuser check for cache update

### CODE REVIEW

File: chrome-ux-report/src/Code.js [hunk 1]
```diff
@@ -324,6 +344,7 @@ function getOriginDataset(request) {
   var origin = {};
   origin.lastUpdate = getDatasetUpdate(request);
   origin.url = validateUrl(request.configParams);
+  origin.url = origin.url.toLowerCase();
```
Comment (diminishedprime) [inline chrome-ux-report/src/Code.js]: aren't some urls case-sensitive?

File: chrome-ux-report/src/Code.js [hunk 2]
```diff
@@ -278,9 +296,11 @@ function flushCache() {
  * @param {number} newDataUpdate The timestamp for last data update in YYYYMMDD format.
  */
 function updateDataUpdateFlag(newDataUpdate) {
-  propStore.set("script", crux.lastDataUpdateFlag, newDataUpdate);
-  console.log("It seems entire BigQuery dataset has been updated");
-  flushCache();
+  if (isAdminUser()) {
+    propStore.set("script", crux.lastDataUpdateFlag, newDataUpdate);
+    console.log("It seems entire BigQuery dataset has been updated");
```
Comment (diminishedprime) [inline chrome-ux-report/src/Code.js]: Nit: phrasing is a bit odd.

File: chrome-ux-report/src/Code.js (hunk 1 above)
Comment (minhaz) [reply chrome-ux-report/src/Code.js]: url's domain part is not case-sensitive. CrUX stores information at domain level.

**Comment by minhaz** [comment]
Updated.

**Comment by diminishedprime** [comment]
Looks good, thanks!

**Pull request merged**